tokio/src/sync/futures.rs

```diff
@@ -0,0 +1,3 @@
+//! Sync future types.
+
+pub use super::notify::Notified;
```

Comment:
How does the documentation for the `Notified` type look?

Reply:
![image](https://user-images.githubusercontent.com/1835307/121728599-2b2bfa00-caf6-11eb-9d87-089fd9de24e2.png)
Like this. (Fixed the link in the last commit)